opencti-platform/opencti-front/src/private/components/analysis/MalwareAnalyses.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { FunctionComponent } from 'react';
+import ListLines from '../../../components/list_lines/ListLines';
+import { usePaginationLocalStorage } from '../../../utils/hooks/useLocalStorage';
+import { Filters } from '../../../components/list_lines';
+import Security from '../../../utils/Security';
+import { KNOWLEDGE_KNUPDATE } from '../../../utils/hooks/useGranted';
+import MalwareAnalysisCreation from './malware_analyses/MalwareAnalysisCreation';
+import useQueryLoading from '../../../utils/hooks/useQueryLoading';
+import MalwareAnalysesLines, { malwareAnalysesLinesQuery } from './malware_analyses/MalwareAnalysesLines';
+import { MalwareAnalysesLinesPaginationQuery, MalwareAnalysesLinesPaginationQuery$variables } from './malware_analyses/__generated__/MalwareAnalysesLinesPaginationQuery.graphql';
+import { MalwareAnalysisLineDummy } from './malware_analyses/MalwareAnalysisLine';
+
+const MalwareAnalyses: FunctionComponent = () => {
+  // TODO generate necesssary Objects
+  const { viewStorage, helpers, paginationOptions } = usePaginationLocalStorage<MalwareAnalysesLinesPaginationQuery$variables>('view-malware-analyses', {
+    searchTerm: '',
+    sortBy: 'submitted', // TODO correct
+    orderAsc: false,
+    openExports: false,
+    filters: {} as Filters,
+    numberOfElements: {
+      number: 0,
+      symbol: '',
+    },
+  });
+
+  const renderLines = () => {
+    const {
+      sortBy,
+      orderAsc,
+      searchTerm,
+      filters,
+      openExports,
+      numberOfElements,
+    } = viewStorage;
+    const dataColumns = {
+      result_name: {
```

Comment:
In order to get more readable/generic XXXLine components, I introduced a render attribute in dataColumns, can you check on it ?